commerce_coordinator/apps/commercetools/management/commands/create_commercetools_product.py

```diff
@@ -0,0 +1,66 @@
+import json
+
+from commercetools.platform.models import (
+    ProductDraft,
+    ProductPriceModeEnum,
+    ProductTypeResourceIdentifier,
+    ProductVariantDraft
+)
+
+from commerce_coordinator.apps.commercetools.management.commands._ct_api_client_command import (
+    CommercetoolsAPIClientCommand
+)
+
+product_json = '''
+
+
+
+'''
+
+
+class Command(CommercetoolsAPIClientCommand):
+    help = "Create a commercetools product from a JSON string or file"
+
+    def handle(self, *args, **options):
+        if product_json:
```

Comment:
I am not sure if this will work. I am not sure why we need this command., For CT dev-01, I guess?

Reply:
no this is for debugging in prod since we dont have rights to create via MC. This is only when in extreme cases, we can't find why denodo creation job is fialing on json.